pkgs/applications/misc/keepmenu/default.nix

```
@@ -0,0 +1,28 @@
+{ lib, python3Packages, python3 }:
+
+with python3Packages;
+
+buildPythonApplication rec {
+  pname = "keepmenu";
+  version = "1.2.2";
+  disabled = pythonOlder "3.7";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "SeVNtONH1bn2hb2pBOVM3Oafrb+jARgfvRe7vUu6Gto=";
+  };
+
+  propagatedBuildInputs = [
+    pykeepass
+    pynput
+  ];
+
+  doCheck = false;
```

Why are those disabled?

```suggestion
  doCheck = false;

  pythonImportsCheck = [ "keepmenu" ];
```